Week1/solution/ALL.py

```diff
@@ -0,0 +1,56 @@
+choice=1
+while(choice!=6):

```

Comment:
can you write this condition some other way...!!!

Reply:
yes, there are many way to make the  menu driven program in python
let asssume 
choice = 0 
while(choice!=6):

or many other way to handle the loop its upon us how we manage the loop